linux-tools/sqlite/sqlite.sh

```diff
@@ -64,6 +65,7 @@ function test03()
 {
 	tc_register "Sqlite3 pivot and attach test"
         $test_dir/runpivot.sh > $TCTMP/test03.out 2>$stderr
+	cp $TCTMP/test03.out /home/logs
 	tc_fail_if_bad $? "Unexpected response from sqlite3 command" || return
```

Comment:
cp $TCTMP/test03.out /home/logs - Is this required or for debugging this was added ?

Should it be as below ?  : 

$test_dir/runpivot.sh > $TCTMP/test03.out 2>$stderr
tc_fail_if_bad $? "Unexpected response from sqlite3 command" || return